### DISCUSSION TITLE
noob script error i guess

### DISCUSSION THREAD

**Discussion by piar22** [blizzhackers/pickits#13] [Q&A about pickit codes]
[name] == grandmatronbow && [quality] == <= superior # [bowandcrossbowskilltab] <= 3 && [sockets] == 4 && [enhanceddamage] <= 15

tell me what is my error

**Comment by mf022** [comment]
>  [quality] == <= superior

... <= ... is enough
<https://github.com/blizzhackers/pickits/blob/master/NipGuide.md/#comparison-symbols>

**Comment by piar22** [comment]
thx sir
